bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/GNIRS_BP.txt

```diff
@@ -60,11 +60,16 @@ INCLUDE {5}, {6}, {12}-{14} in a target-specific Scheduling Group
 # Short Camera bad-pixel patch at +5.25 < Q < +11.25 -> Use -2,+4 and -4,+2
 # Long Camera bad-pixel patch at  +1.75 < Q <  +3.75 -> Use -1,+5 for both sci and std
 IF CROSS-DISPERSED == No:
-   IF PI Central Wavelength < 2.5um:
-      SET Q-OFFSET to -2, 4, 4, -2 respectively IN ITERATOR CALLED 'ABBA offset pattern' for {12}          # Science
-      SET Q-OFFSET to -4, 2, 2, -4 respectively IN ITERATOR CALLED 'ABBA offset pattern' for {6}, {14}     # Tellurics
-   ELSE:
-      SET Q-OFFSET to -1, 5, 5, -1 respectively IN ITERATOR CALLED 'ABBA offset pattern' for {6},{12},{14} # Science & Tellurics
+  IF pixel scale = 0.15" AND wavelength < 2.5um:
+    SET Q-OFFSET to +2, -4, -4, +2 IN ITERATOR CALLED 'ABBA offset pattern' FOR {12} # Science
+    SET Q-OFFSET to -2, +4, +4, -2 IN ITERATOR CALLED 'ABBA offset pattern' FOR {6},{14} # Telluric
+  ELIF pixel scale = 0.15" AND wavelength >= 2.5um:
+    SET Q-OFFSET to +2, -4, -4, +2 IN ITERATOR CALLED 'ABBA offset pattern' FOR {6},{12},{14} # Sci & Tell
+  ELIF pixel scale = 0.05" AND wavelength < 2.5um:
+    SET Q-OFFSET to -1, +5, +5, -1 IN ITERATOR CALLED 'ABBA offset pattern' FOR {12} # Science
+    SET Q-OFFSET to +1, -5, -5, +1 IN ITERATOR CALLED 'ABBA offset pattern' FOR {6},{14} # Telluric
+  ELIF pixel scale = 0.05" AND wavelength > 2.5um:
+    SET Q-OFFSET to -1, +5, +5, -1 IN ITERATOR CALLED 'ABBA offset pattern' FOR {6},{12},{14} # Sci & Tell
```

Comment:
~Are we missing `pixel scale = 0.05" AND wavelength == 2.5um` ?~  I see the code handles this case, but I guess we could update the pseudocode.